smartcontractkit/external-adapters-js
# Bugfix/DF-19188 local ncfx ea crypto endpoint provider indicated time unix ms is incorrect

## Closes #[DF-19188](https://smartcontract-it.atlassian.net/browse/DF-19188)

## Description
Fix `providerIndicatedTimeUnixMs`, which is currently dependent on timezone and causing log spam for local NCFX EAs (and presumably EAs deployed with clocks anywhere other than UTC).

## Changes

- Add `Z` if missing to timestamp
- Related change in test to imitate real data (See https://docs.newchangefx.com/#d56e1fe3-87fb-46e2-8ea0-1f2e43161f9e for details)

## Steps to Test

1. Run NCFX EA with `yarn start`
2. Make a crypto request
3. Ensure no log spam
4. Sanity check `providerIndicatedTimeUnixMs` for correct time

## Quality Assurance



[DF-19188]: https://smartcontract-it.atlassian.net/browse/DF-19188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
